docs/notebooks/data-designer/data-designer-101/2-structured-outputs-and-jinja-expressions.ipynb

```diff
@@ -33,14 +33,25 @@
    "outputs": [],
    "source": [
     "%%capture\n",
-    "%pip install git+https://github.com/gretelai/gretel-python-client"
+    "%pip install -U gretel_client"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Found cached Gretel credentials\n",
+      "Logged in as kirit.thadaka@gretel.ai ✅\n",
+      "Using project: default-sdk-project-1b613ec72030408\n",
+      "Project link: https://console-eng.gretel.ai/proj_2uY0cfM0kjiegpyEZvCHNKZYxGf\n"
```

Comment:
can you clear the outputs on this notebook?

Reply:
Done